sdk/servicebus/service-bus/package.json

```diff
@@ -56,7 +56,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"samples/**/*.{ts,js}\" \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy=true -- --timeout 600000 --full-trace \"dist-esm/test/internal/**/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\"",
+    "integration-test:node": "c8 mocha --loader=../../../common/tools/esm4mocha.mjs -r source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml --full-trace --timeout 600000 --full-trace --full-trace \"dist-esm/test/internal/**/*.spec.js\" \"dist-esm/test/public/**/*.spec.js\"",
```

Comment:
can you add a reference to how this works in the PR description? with regards to c8/mocha/loader, etc

Reply:
The difference between what dev-tool runs and this is
```diff
--r ../../../common/tools/esm-workaround -r esm
+--loader=../../../common/tools/esm4mocha.mjs
```

Reply:
I updated PR description. Hope it makes more sense now.